roles/common/tasks/endpoint_tests.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: Get Endpoint
+  ansible.builtin.shell:
+    cmd: |
+      oc project openstack     
+      kubectl exec openstackclient -- openstack endpoint list --service="{{ item[0] }}" --service="{{ item[1] }}"  --interface="{{ item[2] }}"
```

Comment:
You can use ``oc rsh openstackclient openstack endpoint list ....``

the `-n openstack` should be included to specify the project/namespace